README.md

```diff
@@ -2,14 +2,14 @@
 
 > Welcome to Open Source
 
-📅 Saturday, February 25, 2017 (1:00 PM to 3:00 PM)  
-📍 Ackerman, UCLA  
```

Comment:
can you please not remove the trailing spaces? They are needed for line breaks :) If you use `.editorconfig` you can add this:

```
[*.md]
trim_trailing_whitespace = false
```